### PR TITLE
Clean up

### LINKED ISSUE #5: Dev dataset not needed
Just an issue I noticed, whoever wants to take it may.

In our test train split we are also splitting out a dev dataset. The dev dataset makes sense to do when we are training over multiple epochs and want to prevent overfitting. It gives us an "runtime" estimation of the performance of the network without having to go through the whole test set. If you want more explanation of why we don't need a dev dataset, reach out to me here or comment on this issue.

So I want to cut out the dev dataset and just have train / test with an 80 / 20 split. However this is going to break a bunch of code in Logistic_reg.py and bayes.py because certain functions here are expecting there to be 3 datasets.

### PR DESCRIPTION
**Pull request by NoahSchiro**
Hello all. This is just some small changes + pulling it all together

- [x] Removed extra training datasets
- [x] Removed dev set (only applicable to multi-epoch neural nets)
- [x] Calling all the models from main.py
- [x] Renamed files to be short and concise
- [x] Adjusted train test split to be 90/10. I'm not too worried about overfitting because this is not multi-epoch.
- [x] Created a majority rule predictor. This takes the prediction for everything and says that if 2 of the 3 models predict the same POS tag, then predict that. If there is no majority, then just take the output from log reg, which is the best model so far. Accuracy 92.9%

Please pull down and test for yourself. It takes an extraordinarily long time to train and test everything

### CODE REVIEW

**Review by Kofwana — APPROVED** [review]
The increased cohesion is noticeable! I understand your point about the slower training process. One approach to reduce initial training time is to utilize a smaller subset of the training data initially. Once a robust model is established with this subset, we can then try fine-tuning it on the complete dataset. Also, we may need to add the unlabeled test file into the project soon, following the same inclusion process as the train.txt file in the main script. Keep up the excellent work, Noah! 😊🚀

**Comment by NoahSchiro** [comment]
@Kofwana good point. Another way we could speed up things is parallelization. I was taking a look at my resource monitor as it was training and I think it only uses one CPU core so we could always just spin up multiple threads.

I didn't do this because I am lazy and just want to finish the project.

To your point about the test dataset, we can include that as well and do the testing on that as opposed to the split. If someone else wants to pick that up once this is merged, go for it

**Review by afq5024 — APPROVED** [review]
Ya'll are too smart for me. This looks great! Very cohesive and makes much more sense. I understand the running time issue, but I'm pretty sure we're not gonna be graded on the running time? I think it'd be great to figure out a better and more efficient way, but because of time and other things I'm sure are on our schedule I think this work is great!

**Comment by NoahSchiro** [comment]
Yeah running time doesn't matter as far as the grade goes. Anyhow, merging.
